packages/upgrade/solscripts/Upgrade.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+/*//////////////////////////////////////////////////////////////
+                                 IMPORTS
+//////////////////////////////////////////////////////////////*/
+// Contracts to be upgraded
+import { Home } from "../../contracts-core/contracts/Home.sol";
```

Comment:
should not use relative imports. Should use package-scoped imports

Reply:
this might be a miscommunicated requirement.

Upgrades and deploy should use the npm package artifacts. The goal is to ensure that we ensure we are deploying a known, tagged, and published version. We should not be compiling these on demand for deployment

Reply:
I don't this is possible, given **how** this works. 

The best thing we can do IMHO is to use a proper process:
1) Code free
2) Tag and release
3) git checkout to tag
4) upgrade/deploy

we could package that into a yarn/bash script, so when upgrading it always checks out to the latest tagged version for example

Reply:
https://github.com/nomad-xyz/monorepo/pull/349/commits/4edac98d077650b5f60019c3aed2bbf3a1cb85d0

Reply:
@odyslam and i discussed in DMs, this is indeed possible 😃